app/Main.hs

```diff
@@ -13,12 +13,12 @@ import System.Exit (die, exitFailure)
 import Fmt
```

Comment:
The latest fixup commit will be squashed against `[#16] `BackendName` newtype add`, but it contains changes that relate to 3 different commits. 

This fixup commit should be split accordingly.

Reply:
Done